diagnostic.md

```diff
@@ -7,36 +8,36 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone <git@github.com:kfili/git-basics-diagnostic.git>
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+branch the local repo
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git branch response
```

Comment:
yes, but in addition, we also need to go "git checkout response", or you can combine both with "git checkout -b response"